### ISSUE TITLE
Bug with Android push notification parsing

### ISSUE THREAD

**Issue by kolibrigames-ajimine-vahid** [Appboy/appboy-unity-sdk#58]
https://github.com/Appboy/appboy-unity-sdk/blob/a55e60f1a56547b19c4f514e1f2dc739ae4c9c5d/Assets/Plugins/Appboy/models/PushNotification.cs#L33

This is wrong as the package for Android push notifications is "extra", not "extras"

Here is a suggested change
```
if (json["extra"] != null) {
        Extras = JsonUtils.JSONClassToDictionary(JSON.Parse(json["extra"].Value).AsObject);
```

**Comment by kolibrigames-ajimine-vahid** [comment]
Currently, our work around for this is the following
```
private static string ApplyAndroidFix(string message)
{
  var json = JSON.Parse(message);
  json["extras"] = JSON.Parse(json["extra"].Value);
  return json.ToString();
}
```

**Comment by Bucimis** [comment]
@kolibrigames-ajimine-vahid thanks for reporting, can confirm that this is an issue. We'll look into prioritizing a fix

**Comment by radixdev** [comment]
Hi @kolibrigames-ajimine-vahid ,

This will be included in the next release, thank you for your patience.

**Comment by radixdev** [comment]
Hi @kolibrigames-ajimine-vahid ,

This is fixed in 2.2.2, released just now
